### PR TITLE
chore: don't require approvals for any updates

### PR DESCRIPTION
**Pull request by rarkins**

### CODE REVIEW

**Comment by rarkins** [comment]
Also removes `ignorePaths` so that test Dockerfiles are Renovated

**Review by viceice — COMMENTED** [review]
Ok, we need to disable major and minor automerge on tests. We want to test old versions

**Comment by rarkins** [comment]
Should I disable automerge completely until we are sure?

**Comment by viceice** [comment]
> Should I disable automerge completely until we are sure?

`test/` path should be enough

File: renovate.json [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
-  "extends": ["github>renovatebot/presets-internal:docker-images"]
+  "extends": ["github>renovatebot/presets-internal:docker-images"],
+  "ignorePaths": [],
+  "packageRules": [{
+    "description": "Don't hold back any PRs for approval",
+    "packagePatterns": ["*"],
+    "dependencyDashboardApproval": false
+  }, {
+    "description": "Don't automerge test minor/major because we might want to test old versions too",
+    "paths": ["test/"],
```
Comment (viceice) [inline renovate.json:11]:
```suggestion
    "updateTypes": ["major", "minor"],
```
restrict update types?

Comment (rarkins) [reply renovate.json:11]: ok, will need 2 rules

**Review by viceice — APPROVED** [review]

**Pull request merged**